src/loops/simple_programs_with_repitition/easy/q4/ReverseReverseTest.java

```diff
@@ -0,0 +1,39 @@
+package loops.simple_programs_with_repitition.easy.q4;
+
+import global.variables.*;
+import global.variables.clauses.*;
+import global.BaseTest;
+
+public class ReverseReverseTest extends BaseTest {
+
+    int input = 9268236;
+
+    public Clause[] testSentence() {
+        
+        Clause[] output = new Clause[4];
```

Comment:
This way looks more confusing than just doing the basic pattern. Since we're not looping, let's do this the normal way